### PR TITLE
Scribe refactorization

### PR DESCRIPTION
**Pull request by rsevilla87**
A bunch of changes.

- Remove scribe_uuid field, we're not using it and makes each indexed document unique. If we want to avoid indexing duplicated documents, we must get rid of it.
- Fix all linting issues.
- Improve error handling
- Fix and simplify CPU vulnerabilities module
- Add new modules: kube-apiserver, kube-controllermananager, default-ingress-controller
- Extract only some fields such as "spec" and metadata.name from several k8s and ocp modules to avoid indexing useless information such as metadata
- Remove `__init__` method from modules inheriting from ScribeModuleBaseClass as is no longer required with python3.

Depends on https://github.com/cloud-bulldozer/stockpile/pull/112

Example of the output of the k8s and openshift roles after this PR: https://termbin.com/0m6r

Signed-off-by: Raul Sevilla <rsevilla@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,12 +209,11 @@ would involve:
   class Scribe_module_1(ScribeModuleBaseClass):
 
       def __init__(self, input_dict=None, module_name=None, host_name=None,
-                   input_type=None, scribe_uuid=None):
```
Comment (jtaleric) [inline README.md:212]: What if someone isn't using our work?

**Review by dry923 — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (jtaleric) [reply README.md:212]: All documents have a inherit `_id` too, right?

Comment (rsevilla87) [reply README.md:212]: If you mean UUID, stockpile-wrapper takes care of adding an UUID to the documents.
https://github.com/cloud-bulldozer/bohica/blob/master/stockpile-wrapper/stockpile-wrapper.py#L64

pd: I have also updated some of the docs.

Comment (jtaleric) [reply README.md:212]: ES has a `_id`
```
  "_id": "4538dbdefd127dabecb3a6fde17f828978a5245cc5e9e5525362ddbc6059d37d",
  "_version": 1,
  "_score": null,
  "_source": {
    "module": "cpuinfo",
    "host": "localhost",
    "source_type": "stockpile",
    "scribe_uuid": "9c368a0c-0156-473b-9e29-8fcd801046ca",
    "value": {
      "Architecture": "x86_64",
      "CPU op-mode(s)": "32-bit, 64-bit",
      "Byte Order": "Little Endian",
      "CPU(s)": "8",
```

Comment (rsevilla87) [reply README.md:212]: yeah, that _id is now generated by stockpile-wrapper https://github.com/cloud-bulldozer/bohica/blob/master/stockpile-wrapper/stockpile-wrapper.py#L65

Comment (jtaleric) [reply README.md:212]: ack!

**Comment by jtaleric** [comment]
LGTM!

**Pull request merged**